pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -78,6 +78,16 @@
  * Main class for Pulsar broker service
  */
 public class PulsarService implements AutoCloseable {
+
+    private static final int EXECUTOR_THREAD_COUNT = Integer
+            .parseInt(System.getProperty("pulsar.executor-thread-count", "20"));
```

Comment:
instead of reading from `System-property` what if we can make it configurable via `ServiceConfiguration` so, broker can be started with configured number of threads which may depend on host hardware or resource usage.?